geo/src/main/res/layout/geopoint_dialog.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This layout is mostly stolen from `progress_dialog.xml`. As we've talked about revising this interface (and I wanted to move the layout to another module) I thought it made sense to just create a new layout for this.